display_manager.py

```diff
@@ -197,11 +198,19 @@ def _render_entities(cls):
         entities_sorted = sorted(cls.dungeon.current_level.entities,
                                  key=lambda x: x.render_priority.value)
         for entity in entities_sorted:
+            # Draw visible entities
             if cls.dungeon.current_level.fov[entity.pos]:
                 cls.console.draw_char(
                     entity.pos.x, entity.pos.y, entity.char, entity.color,
                     bg=None
                 )
+            # Remember stairs location
+            if (isinstance(entity, StairsUp) or isinstance(entity, StairsDown)) \
```

Comment:
`isinstance` second arg can take a tuple of classes so you can simplify with `isinstance(entity, (StairsUp, StairsDown))`